### PR TITLE
Python3 compat module.

### PR DESCRIPTION
**Pull request by rudaporto**
* Added _compat module and update python3 fixes to use it. (rudaporto)

### CODE REVIEW

File: src/AccessControl/SecurityManagement.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def setSecurityManager(manager):
 # AccessControl.Implementation inserts SecurityManager.
 
 try:
-    from thread import get_ident
+    from AccessControl._compat import get_ident
```
Comment (tseaver) [inline src/AccessControl/SecurityManagement.py]: The `try: ... except:` should go away, as `_compat` makes it work unconditionally.

File: tox.ini [hunk 2]
```diff
@@ -2,7 +2,6 @@
 envlist =
     # flake8,
     py27,
-    py34,
```
Comment (tseaver) [inline tox.ini]: Why drop 3.4 support?  It remains supported by the Python community until May 2019 (5 years after the 3.4.0 final release).

File: src/AccessControl/SecurityManagement.py (hunk 1 above)
Comment (rudaporto) [reply src/AccessControl/SecurityManagement.py]: Done.

File: tox.ini (hunk 2 above)
Comment (rudaporto) [reply tox.ini]: Fixed.

**Comment by rudaporto** [comment]
@tseaver pull request updated.

**Pull request merged**